tests/mdev_configuration_allocation_test.go

```diff
@@ -20,7 +20,8 @@ import (
 	v1 "kubevirt.io/api/core/v1"
 	"kubevirt.io/client-go/kubecli"
 
-	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+	"kubevirt.io/kubevirt/pkg/virt-config/featuregate"
+
```

Comment:
nit: remove new line

Reply:
The newline separates production from e2e tests packages.